docs/sources/developers/plugins/add-support-for-explore-queries.md

```diff
@@ -85,25 +85,20 @@ The query editor for Explore is similar to the query editor for the data source
    };
    ```
 
-## Support multiple Explore modes
+## Selecting preferred visualisation
 
-Explore lets you query any data source, regardless of whether it returns metrics or logs. You can change which type of query you want to make, by setting the _Explore mode_.
+Explore should be default select a reasonable visualisation for your data so users do not have to tweak and play with the visualisations and just focus on querying. This usually works fairly well and Explore can figure out whether the returned data is time series data or logs or something else.
```

Comment:
```suggestion
Explore should be the default. Select a reasonable visualization for your data so users do not have to tweak and play with the visualizations and just focus on querying. This usually works well and Explore can figure out whether the returned data is time series data or logs or something else.
```

Reply:
Ah that should have been `Explore should by default select` :D